extra-dev/packages/coq-hott/coq-hott.8.13.dev/opam

```diff
@@ -20,5 +18,5 @@ dev-repo: "git+https://github.com/HoTT/HoTT.git"
 synopsis: "The Homotopy Type Theory library"
 tags: [ "logpath:HoTT" ]
 url {
-  src: "git+https://github.com/HoTT/HoTT.git#V8.13"
+  src: "git+https://github.com/HoTT/HoTT.git#V8.13.1"
```

Comment:
For a release package please use a .tar.gz file - this is more efficient than git.

Reply:
Technically this isn't a release package btw, not that it matters.